README.md

```diff
@@ -1,2 +1,47 @@
-# opa
-A library of policy algorithms deployed as an agent to policy-enable any product or service
+# OPA: Open Policy Agent
+
+OPA is an open source project which can help policy enable any application.
+
+## What is Policy?
+
+Policy defines expected behavior in response to specific events within an application.
+
+There are many examples of functionality within applications which can benefit from rich policy control, e.g, API authorization, VM and container placement, auto-scaling, auto-healing, etc.
+
+The approach to policy can vary greatly within and across applications. For example, policy is often written in natural language (English), stored in documents such as wiki pages, and enforced manually by the administrators of the the application. 
+
+When the cost of manually maintaining and enforcing policy is too high, policy is often defined in way that computers can understand: the application's implementation is updated to include both the definition of policy and the means for enforcing it. Because most applications are implemented in **imperative languages** (such as Java or Python) the policy definition tends to focus on *how* the policy is enforced rather than *what* the expected behavior or state should be. This makes it hard for people unfamiliar with the application's implementation to know what the expected behavior or state should be.
```

Comment:
How about adding... It makes it hard to adapt that policy without rebuilding, retesting, and redeploying the code.  It is harder to have multiple systems understand and enforce that same policy.